src/Stratis.Bitcoin.Features.SmartContracts/Consensus/SmartContractPoABlockHeader.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NBitcoin;
+using NBitcoin.Crypto;
+
+namespace Stratis.Bitcoin.Features.SmartContracts.Consensus
+{
+    public class SmartContractPoABlockHeader : SmartContractBlockHeader
```

Comment:
Ideally this would inherit from PoABlockHeader and add the SC header fields in, but because it uses the base serialization to get the hash right now, it's much easier to have the PoA fields in the "most" derived class